.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         submodules: true
     - if: ${{ runner.os == 'Linux' }}
       run: if [ $(goimports -l .) ]; then goimports -d .; echo 'Failed the goimports format check. Please format the code using "goimports -w ."'; exit 1; fi
-    - run: go test -mod=mod -coverprofile=covprofile ./...
+    - run: go test -mod=mod -coverpkg="./apps,./confgenerator" -coverprofile=covprofile ./...
```

Comment:
confgenerator_test is the only package with tests and it only depends on confgenerator and apps -- adding the other directories might give a misleading coverage result, depending on how you look at it. Although now I'm realizing I missed confgenerator/fluentbit and confgenerator/otel as transitive dependencies anyway.

I don't feel strongly about it; having all directories in the coverage can be useful too. I'll change it to `./...`.

Reply:
I think that means you can omit `-coverpkg` entirely, no?

Reply:
I had initially thought that too, but if you omit it then it will only cover the code in packages with tests, i.e. only confgenerator/ and not apps/ or the others (I had an earlier commit like this and the coverage rate tanked).